cypress/components/flat-table/flat-table.cy.tsx

```diff
@@ -2540,10 +2540,7 @@ context("Tests for Flat Table component", () => {
         }
       }
 
-      flatTablePageSelectNext()
-        .find("button")
-        .focus()
-        .trigger("keydown", keyCode("Space"));
+      flatTablePageSelectNext().find("button").focus().realPress("Space");
```

Comment:
**comment** (non-blocking): Just letting you know this file is currently being refactored Playwright, so there is a chance these changes may have to be re-made, however hopefully during said refactor we will use the proper and preferred keydown triggers/tests

Reply:
thanks for the advice. I'll make the necessary changes if the refactor is merged first.